scripts/core/auth/auth.ts

```diff
@@ -21,6 +21,13 @@ window.fetch = (...args) => {
         return originalFetch(resource, config);
     }
 
+    if (appConfig.testing) {
```

Comment:
It'd be good to avoid `if(testing)` code. I've commented it out and tests still pass so let's drop it.

If dropping it wasn't an option we could look into overwriting fetch in test code rather than here. [reference](https://github.com/facebook/create-react-app/issues/967#issuecomment-256358738)